### ISSUE TITLE
budgie-desktop and rhythmbox freezing when d-bus interface plugin is enabled

### ISSUE THREAD

**Issue by CrisBRM** [budgie-desktop/budgie-desktop#794]
OS is up-to-date Solus.

Music library is "big" (about 200 songs), and using the default settings, meaning the MPRIS D-BUS interface plugin is enabled. Switching between songs causes budgie-desktop (namely, the top and side toolbar) to freeze. Song keeps playing, but both budgie-desktop and rhythmbox remain unresponsive for 15 seconds. 

Disabling the MPRIS D-BUS interface plugin fixes the issue.

**Comment by Womagrid** [comment]
Having the same issue here.  I could be wrong but I believe that the D-BUS interface is used to send notifications on track change and also powers the sidebar widget.

I just found this in my syslog...

rhythmbox[27520]: Failed to send notification (Amazing): Timeout was reached

**Comment by DerRidda** [comment]
This is very much still an issue. Can we get any information if and when this is being tackled?

**Comment by virse** [comment]
Hey there,

same problem in UbuntuBudgie with Audacious as Mediaplayer as well. Still freezing for about 10secs. Disabling the notifications plugin within Audacious helps.

greets, virse

**Comment by sammypanda** [comment]
im not sure if it's related to the d-bus interface still, but yeah, this issue is still a problem now it freezes my budgie up for like 10-45 seconds sometimes it even gets stuck and i need to restart my computer.

**Comment by JoshStrobl** [comment]
Yes it freezes for 25 seconds which is the D-Bus timeout length. I've looked into addressing it but haven't come up with a solution yet.
